Automattic/jetpack
# Added css to hide recommended posts from print post view.

Fixes https://github.com/Automattic/jetpack/issues/16205
Fixes #13005

Jetpack should automatically hide extraneous elements like Likes, Sharing, and Related posts in print media. These are interactive elements and don't make sense in the context of a printed page.

In this PR, we are going to hide the "related posts" section in the "print" view. 

### Screenshots
Consider the below image, this is how related posts look as of now. 

<img width="1908" alt="Screenshot 2021-12-03 at 4 14 11 PM" src="https://user-images.githubusercontent.com/6594561/144589882-f1e3a840-5b32-4803-a239-3b3fff4fa2a8.png">

If I were to print it: 
<img width="695" alt="Screenshot 2021-12-07 at 8 02 59 PM" src="https://user-images.githubusercontent.com/6594561/145048144-b6413ae6-cebb-4092-a203-b629d693520f.png">


This PR introduces the changes to make the print view look like this

<img width="567" alt="Screenshot 2021-12-07 at 8 03 37 PM" src="https://user-images.githubusercontent.com/6594561/145048317-8db40e15-f745-4861-a4fa-51148b2dade9.png">




#### Changes proposed in this Pull Request:

* Added a CSS media query that hides the CSS class `.jp-relatedposts` in the print view. of the file  `projects/plugins/jetpack/modules/related-posts/related-posts.css` 

I add to add the `!important` to the css query for this class to override other rules being set before. Another option I considered was to introduce a new css parent container class, but I thought it added unnecessary overhead. 

Let me know if you have other ideas besides the `!important` solution which I have incorporated. 

<img width="1239" alt="Screenshot 2021-12-03 at 4 02 10 PM" src="https://user-images.githubusercontent.com/6594561/144590571-d4bbce05-bada-46c1-a088-b09f2469a7c9.png">


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
 - Go to Jetpack->Settings->Traffic.
 - Enable related posts. 
 - You can either manually create posts or use some other means to create your posts. I used the `FakePress` plugin to create test data and applied the same tag on the posts that I wanted to be shown as related to each other. 
 - Go to any of the posts' page and observe the related posts section. 
 -  Now, print the page and in the print view, ensure that the related posts section doesn't show up. 
 - To test Dotcom changes, use a public WordPress.com site and sandbox it.
